projects/packages/publicize/src/class-connection-fields.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * Publicize Connection Fields class.
+ *
+ * @package automattic/jetpack-publicize
+ */
+
+namespace Automattic\Jetpack\Publicize;
+
+/**
+ * Publicize Connection Fields class.
+ */
+class Connection_Fields {
```

Comment:
The methods in this class are copied from Publicize_Base class which we can later on clean up to these methods as fallback after deprecating them.